packages/host/tests/helpers/mock-matrix.ts

```diff
@@ -43,16 +63,18 @@ class MockUtils {
   };
 }
 
-// When also using setupBaseRealm, this must come before setupBastRealm so it
-// can pre-fill the local storage mock before services start initializing.
 export function setupMockMatrix(
   hooks: NestedHooks,
   opts: Config = {},
 ): MockUtils {
-  let ownerContainer: Owner[] = [];
+  let testState: { owner?: Owner; sdk?: MockSDK } = {
```

Comment:
Getters with errors?

Reply:
We can leave this as a future TODO. Merging.